fixtures/docstring/tests/bindings/test_docstring.py

```diff
@@ -0,0 +1,54 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Test namespace
+import uniffi_docstring
+assert uniffi_docstring.__doc__
+
+from uniffi_docstring import *
+
+# Test function
```

Comment:
I added checking against specific values, but I wasn't sure how I should add checking for none. Should I just create another set of types without docstrings?

Reply:
I think exactly 1 type without a docstring, then check `__doc__ is None` for that type. When first reading this patch I thought that was going to fail - I then saw that it would work, but to help guard against future refactoring it seems worthwhile.